lab_3/main.py

```diff
@@ -116,7 +223,30 @@ def extract_n_grams(self, encoded_corpus: tuple) -> int:
             )
         )
         """
-        pass
+
+        ngrammed_text = []
+        if not isinstance(encoded_corpus, tuple):
+            return 1
+        for sentence in encoded_corpus:
+            if not isinstance(sentence, tuple):
+                return 1
+            ngrams_in_sentence = []
+            for word in sentence:
+                if not isinstance(word, tuple):
+                    return 1
+                ngram_word = []
+                idk_but_helped = self.size
```

Comment:
Why do you need a separate identical variable?

Reply:
Sorry, I don't understand. Which ones are identical?

Reply:
I also do not know why get_top_k_n_grams does not return anything. Please, help.